documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -351,4 +353,232 @@ Control characters are not allowed in a specification.
 
 **Examples**: A list of control characters in unicode can be found [here](https://unicode-table.com/en/).
 
-Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
\ No newline at end of file
+Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
+
+### <a name="S2008" />S2008 MutabilityWithReadOnly
+**Output Message**:  When property is modeled as "readOnly": true then x-ms-mutability extension can only have "read" value. When property is modeled as "readOnly": false then applying x-ms-mutability extension with only "read" value is not allowed. Extension contains invalid values: '{0}'
```

Comment:
There is no documentation available for it I have raised an issue here https://github.com/Azure/azure-rest-api-specs/issues/1303